tests/test_vembrane.py

```diff
@@ -42,8 +42,15 @@ def test_filter(testcase):
         config = yaml.load(config_fp, Loader=yaml.FullLoader)
 
     vcf = VariantFile(path.joinpath("test.vcf"))
+
     if "raises" in config:
-        exception = getattr(errors, config["raises"])
+        exc = config["raises"]
+        try:
+            exception = getattr(errors, exc)
+        except AttributeError:
+            import builtins
```

Comment:
It's fine to import `builtins` unconditionally on the toplevel.

Reply:
But it's also fine where it is at the moment, right? :D

Reply:
Sure ;D.
(I usually just put inconspicuous imports like `sys`/`os`/`builtins` up top and make slow-to-import or side effect causing ones conditional ;).)